### ISSUE TITLE
Enable alt text for thumbnail images

### ISSUE THREAD

**Issue by elynema** [avalonmediasystem/avalon#5456]
# Description

For first pass, we have made all thumbnail images decorative in Avalon with alt= ''. There are times, however, when a good description of the thumbnail image could be useful to visually impaired users. 

This would need to be added to the "create thumbnail" functionality in the user interface to prompt users for alt text (with at least one or two examples of useful alt text). However, thumbnails are created by default when media is uploaded, so we will continue to have MANY thumbnails with no alt text. This means that we would only want to include alt text for images when it has been generated by users (otherwise, the alt text should remain empty to mark the image as decorative).

The <a /> links that contain these thumbnail images have been moved out of tab context and set as aria hidden / presentation so that they are not found by the screen reader (screen reader will just use the item title to navigate to the item). If we are displaying alt text for some thumbnail images, we may need to make sure the images can be reached by tab navigation in this context.

**Comment by elynema** [comment]
Duplicate of #5487
